components/utilities/headerLink.js

```diff
@@ -31,17 +31,44 @@ const HeaderLink = ({ name, level, className, children }) => {
   const Header = `h${level}`;
   return (
     <>
-      <a name={hash} className="not-link anchor" />
+      <a
+        name={hash}
+        className="
+          no-underline
+          block
+          h-0 w-0
+          overflow-hidden
+          visibility-hidden
+          pointer-events-none
+        "
+      />
 
-      <Header className={classNames("quick-link", className)}>
+      <Header className={classNames("flex items-center", "group", className)}>
         {children}
 
         {copied ? (
-          <div className="status-text">
+          <div
+            className="
+              ml-4
+              text-xs font-normal
+              opacity-70
+              flex items-center
```

Comment:
Move to CSS

Reply:
Fixed on ce224f2d8e897723e6890f8a1ba356373bb1dd93